snapcraft.yaml

```diff
@@ -18,39 +18,38 @@ slots:
 
 apps:
   gnome-2048:
-    extensions: [gnome-3-38]
+    extensions: [gnome]
     command: usr/bin/gnome-2048
     desktop: usr/share/applications/org.gnome.TwentyFortyEight.desktop
     common-id: org.gnome.TwentyFortyEight
+    environment:
+      DISABLE_WAYLAND: 'true'
 
 parts:
   gnome-2048:
     source: https://gitlab.gnome.org/GNOME/gnome-2048.git
     source-type: git
-    source-tag: 3.38.2
+    source-tag: '3.38.2'
     plugin: meson
-    meson-parameters: [--prefix=/snap/gnome-2048/current/usr]
+    meson-parameters:
+      - --prefix=/usr
+      - --buildtype=release
     parse-info: [usr/share/metainfo/org.gnome.TwentyFortyEight.appdata.xml]
     override-pull: |
-      snapcraftctl pull
-      snapcraftctl set-version $(git describe --tags --abbrev=10)
+      craftctl default
+      craftctl set version=$(git describe --tags --abbrev=10)
+      patch -p1 < $CRAFT_PROJECT_DIR/patches/patch1.diff
```

Comment:
Please rename the patch to something more intuitive and use *.diff

Reply:
Done.